src/processor/CodeSystemProcessor.ts

```diff
@@ -31,11 +32,43 @@ export class CodeSystemProcessor {
       ...CaretValueRuleExtractor.processResource(input, fisher, input.resourceType, config)
     );
     input.concept?.forEach((concept: any) => {
-      newRules.push(new ExportableConceptRule(concept.code, concept.display, concept.definition));
+      newRules.push(...this.extractConceptRules(concept, input.name || input.id, fisher));
     });
     target.rules = compact(newRules);
   }
 
+  static extractConceptRules(
+    concept: any,
```

Comment:
I think we could use the `fhirtypes.CodeSystemConcept` type here that we've nicely defined in SUSHI. Everything should still work fine, but we just get some of the type checking advantages in the future.